app/models/conversation.rb

```diff
@@ -86,7 +86,7 @@ def subscribers(user)
   end
 
   def receive(user, person)
-    cnv = Conversation.find_or_create_by!(self.attributes)
+    cnv = Conversation.create_with(self.attributes).find_or_create_by!(self.attributes[:guid])
```

Comment:
Doesn't this have to be .`.find_or_create_by!(guid: guid)` then? This just seems to insert a `WHERE 123456abcdef` into the query if I'm following correctly.